src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  padding-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+}
+
+.stars--0 :nth-child(0) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 :nth-child(1) {
```

Comment:
Avoid using tag names for styling. Instead of using ':nth-child(1)', consider using a class or another selector that doesn't involve tag names.